README.md

```diff
@@ -304,6 +297,8 @@ the next section.
 
 The execution order of task is determined by dependencies that are defined between the tasks via inputs and outputs as well as explicit orders that are defined via `runAfter`.
 
+`workspaces` field allow you to specify one or more volumes that each Task in the Pipeline requires during execution. You specify one or more Workspaces in the `workspaces` field.
```

Comment:
The `workspaces` field allows

Reply:
OK. It was already merged. Will create a new pr for this.